packages/share_plus/share_plus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.1.1
+
+- Add *WithResult methods to get feedback on user action
+
```

Comment:
small thing, I'd prefer if it was `3.2.0` instead. It shouldn't be a breaking change so no need for `4.0.0`

Reply:
Will do as soon as I am home 👍🏻